chann_test.go

```diff
@@ -245,6 +245,7 @@ func TestNonblockSelectRace(t *testing.T) {
 		c1 := chann.New[int]()
 		c2 := chann.New[int]()
 		c1.In() <- 1
+		time.Sleep(time.Millisecond)
```

Comment:
I am not sure whether this is the correct approach to fix the issue. Can you elaborate more what why Sleep can effectively solve the issue?

Reply:
I think the main change is to select c2 instead of c1. sleep is that I wanted to make sure that c1 did already have a value when I tested it and it looked unnecessary.
The original test's failure scenario looks like:
1. c1 has a value
2. writing to c2
3. consume c1 
4. the second goroutine goes into default.

Reply:
Thanks for your review!  💚 💙 💜 💛 ❤️